mmv1/products/metastore/Service.yaml

```diff
@@ -184,6 +189,33 @@ properties:
       - :DEVELOPER
       - :ENTERPRISE
     default_from_api: true
+    conflicts:
+      - scalingConfig
+
+  - !ruby/object:Api::Type::NestedObject
+    name: 'scalingConfig'
+    description: |
+      Represents the scaling configuration of a metastore service.
+    properties:
+      - !ruby/object:Api::Type::Enum
+        name: 'instanceSize'
+        description: |
+          Metastore instance sizes.
+        required: false
+        conflicts:
+          - tier
+        values:
+          - :INSTANCE_SIZE_UNSPECIFIED
```

Comment:
This is not an acceptable value. Could we remove this one?

Reply:
will do.

Reply:
done